### PR TITLE
beefi: Init at 0.1.1

### LINKED ISSUE #108996: beefi
**Project description**
Converts Linux images and kernels into UEFI images.

**Metadata**

* homepage URL: https://github.com/jfeick/beefi
* source URL: https://github.com/jfeick/beefi
* license: GPL3
* platforms: Linux

### PR DESCRIPTION
**Pull request by tu-maurice**
###### Motivation for this change
Closes #108996

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: maintainers/maintainer-list.nix [hunk 1]
```diff
@@ -9299,6 +9299,11 @@
     githubId = 303489;
     name = "Manuel Bärenz";
   };
+  tu-maurice = {
```
Comment (SuperSandro2000) [inline maintainers/maintainer-list.nix:9302]: Please put this into a separate commit with the message ``maintainers: add tu-maurice``.

File: maintainers/maintainer-list.nix [hunk 2]
```diff
@@ -9299,6 +9299,11 @@
...
+    email = "valentin.gehrke@zom.bi";
+    github = "tu-maurice";
```
Comment (SuperSandro2000) [inline maintainers/maintainer-list.nix:9304]:
```suggestion
    github = "tu-maurice";
    githubId = 16151097;
```

File: pkgs/os-specific/linux/beefi/default.nix [hunk 3]
```diff
@@ -0,0 +1,44 @@
+{ stdenv, lib, fetchFromGitHub,
+  binutils-unwrapped,
+  systemd }:
```
Comment (SuperSandro2000) [inline pkgs/os-specific/linux/beefi/default.nix:3]:
```suggestion
{ stdenv
, lib
, fetchFromGitHub
, installShellFiles
, binutils-unwrapped
. systemd }:
```

File: pkgs/os-specific/linux/beefi/default.nix [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+
+  src = fetchFromGitHub {
+    owner = "jfeick";
+    repo = "beefi";
+    rev = version;
+    sha256 = "1180avalbw414q1gnfqdgc9zg3k9y0401kw9qvcn51qph81d04v5";
+  };
+
+  outputs = [ "out" "man" ];
+
```
Comment (SuperSandro2000) [inline pkgs/os-specific/linux/beefi/default.nix:17]:
```suggestion
```

File: pkgs/os-specific/linux/beefi/default.nix [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+  buildInputs = [
+    binutils-unwrapped
+    systemd
+  ];
+
+  buildPhase = ''
+    gzip < beefi.1 > beefi.1.gz
+  '';
```
Comment (SuperSandro2000) [inline pkgs/os-specific/linux/beefi/default.nix:25]:
```suggestion
```

File: pkgs/os-specific/linux/beefi/default.nix [hunk 6]
```diff
@@ -0,0 +1,44 @@
...
+
+  patchPhase = ''
+    substituteInPlace beefi --replace objcopy ${binutils-unwrapped}/bin/objcopy \
+                            --replace /usr/lib/systemd ${systemd}/lib/systemd
```
Comment (SuperSandro2000) [inline pkgs/os-specific/linux/beefi/default.nix:29]:
```suggestion
    substituteInPlace beefi \
      --replace objcopy ${binutils-unwrapped}/bin/objcopy \
      --replace /usr/lib/systemd ${systemd}/lib/systemd
```

File: pkgs/os-specific/linux/beefi/default.nix [hunk 7]
```diff
@@ -0,0 +1,44 @@
...
+  '';
+
+  installPhase = ''
+    install -Dm644 beefi.1.gz $man/share/man/man1/beefi.1.gz
```
Comment (SuperSandro2000) [inline pkgs/os-specific/linux/beefi/default.nix:33]:
```suggestion
    installManPage beefi.1
```

**Comment by SuperSandro2000** [comment]
This is a semi-automatic executed nixpkgs-review which does not build all packages (e.g. lumo, tensorflow or pytorch)
If you find some bugs or got suggestions for further things to search or run please reach out to SuperSandro2000 on IRC.

Result of `nixpkgs-review pr 109329` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
<summary>
1 package built:
</summary>
<ul>
<li>beefi</li>
</ul>
</details>

**Comment by SuperSandro2000** [comment]
Please do not rebase against master if it is not required.

**Comment by SuperSandro2000** [comment]
This is a semi-automatic executed nixpkgs-review which does not build all packages (e.g. lumo, tensorflow or pytorch)
If you find some bugs or got suggestions for further things to search or run please reach out to SuperSandro2000 on IRC.

Result of `nixpkgs-review pr 109329` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
<summary>
1 package built:
</summary>
<ul>
<li>beefi</li>
</ul>
</details>

**Comment by danwdart** [comment]
Thank you!
